openadapt/widget/app.py

```diff
@@ -0,0 +1,128 @@
+import kivy
+from subprocess import Popen
+import signal
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.uix.floatlayout import FloatLayout
+from kivy.core.window import Window
+import pygetwindow as gw
+from loguru import logger
+import ctypes
+from ctypes import wintypes
+from kivy.clock import Clock
+import sys
+import pywinauto
+import Quartz
+
+
+class OpenAdaptWidget(FloatLayout):
+    def __init__(self, **kwargs):
+        super(OpenAdaptWidget, self).__init__(**kwargs)
+        self.window = Window
+        self.window.borderless = True
+        self.window.size = (50, 50)
+        self.window.clearcolor = (255, 255, 255)
+        self.window.always_on_top = True
+        self.PROC = None
```

Comment:
Since this isn't a constant, what do you think about naming it e.g. `process`? Even better, how about splitting it into `replay_process` and `record_process`?

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/290/commits/ecf0e6565d2c2d007ba9aa5445eac976fa8f2a07